### PR TITLE
Performance graph updates for 2.1.0

### PR DESCRIPTION
**Pull request by tzinsky**
This is similar to PRs done in previous releases such as: https://github.com/chapel-lang/chapel/pull/24647.

### CODE REVIEW

File: util/test/perf/perfgraph.js [hunk 1]
```diff
@@ -167,9 +167,12 @@ var branchInfo = [
                     "releaseDate": "2024-03-21",
                     "branchDate" : "2024-03-15",
                     "revision": -1},
+                    { "release": "2.1.0",
+                      "releaseDate": "2024-06-27",
+                      "branchDate" : "2024-06-24",
+                      "revision": -1}
```
Comment (arezaii) [inline util/test/perf/perfgraph.js:173]: this should continue to have a trailing comma, I believe

Comment (arezaii) [reply util/test/perf/perfgraph.js:173]: also this entry is over-indented

**Review by arezaii — CHANGES_REQUESTED** [review]
just the comments about perfgraph.js

File: util/test/perf/perfgraph.js (hunk 1 above)
Comment (tzinsky) [reply util/test/perf/perfgraph.js]: Pushed new changes to resolve these issues.

**Review by arezaii — APPROVED** [review]
thanks!

**Pull request merged**